proxies/events-graphql-proxy/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "events-graphql-proxy",
+  "description": "Events Helsinki GraphQL proxy.",
+  "version": "0.2.10",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf --no-glob ./tsconfig.tsbuildinfo",
+    "build": "NODE_ENV=production webpack --progress --mode production --config config/webpack.config.js",
+    "dev": "cross-env GRAPHQL_PROXY_PORT=4100 nodemon",
+    "start": "node build",
+    "test": "jest",
+    "prepare": "husky install",
+    "pre-commit": "lint-fix",
+    "lint": "eslint . --ext .ts,.tsx,.js,.jsx,.cjs,.mjs,.mdx,.graphql --cache --cache-location ../../.cache/eslint/events-graphql-proxy.eslintcache",
+    "lint-fix": "eslint . --ext .ts,.tsx,.js,.jsx,.cjs,.mjs,.mdx,.graphql --fix",
+    "check-dist": "es-check -v",
+    "typecheck": "tsc --project ./tsconfig.json --noEmit",
+    "generate:graphql": "graphql-codegen",
+    "docker:prune-cache": "docker buildx prune",
+    "docker:install": "docker buildx bake -f docker-compose.proxy.yml --progress=tty deps",
+    "docker:develop": "cross-env DOCKER_BUILDKIT=1 docker-compose -f ./docker-compose.yml up",
+    "docker:develop:reinstall": "cross-env DOCKER_BUILDKIT=1 docker-compose -f docker-compose.yml build --progress=tty --no-cache --force-rm develop",
+    "docker:build": "cross-env DOCKER_BUILDKIT=1 docker-compose -f docker-compose.yml build --progress=tty runner",
+    "docker:serve": "docker-compose -f docker-compose.yml --env-file .env up runner"
+  },
+  "dependencies": {
+    "@apollo/datasource-rest": "4.3.2",
+    "@apollo/server": "4.3.0",
+    "@apollo/subgraph": "2.2.2",
+    "@apollo/utils.keyvaluecache": "2.1.0",
+    "@sentry/integrations": "7.28.1",
+    "@sentry/node": "7.28.1",
+    "@types/graphql-depth-limit": "1.1.3",
+    "@types/node": "18.11.18",
+    "@types/uuid": "9.0.0",
+    "body-parser": "1.20.1",
+    "cors": "2.8.5",
+    "dotenv": "16.0.3",
+    "express": "4.18.2",
+    "graphql": "16.6.0",
+    "graphql-depth-limit": "1.1.0",
+    "graphql-tag": "2.12.6",
+    "lodash": "4.17.21",
+    "promise": "8.3.0",
+    "typescript": "4.9.4",
+    "uuid": "9.0.0",
+    "winston": "3.8.2"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "devDependencies": {
+    "@babel/core": "7.20.7",
+    "@babel/plugin-syntax-flow": "7.18.6",
+    "@babel/plugin-transform-react-jsx": "7.20.7",
+    "@graphql-codegen/cli": "2.16.1",
+    "@graphql-codegen/typescript": "2.8.5",
+    "@graphql-codegen/typescript-operations": "2.5.10",
+    "@graphql-codegen/typescript-resolvers": "2.7.10",
```

Comment:
I don't think that the codegen should be needed. It's for client I suppose.

Reply:
removed

Reply:
removed